contracts/contracts/gatekeepers/zupass/Groth16Verifier.sol

```diff
@@ -0,0 +1,428 @@
+// SPDX-License-Identifier: GPL-3.0
+/*
+    Copyright 2021 0KIMS association.
+
+    This file is generated with [snarkJS](https://github.com/iden3/snarkjs).
+
+    snarkJS is a free software: you can redistribute it and/or modify it
+    under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    snarkJS is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+    or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+    License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with snarkJS. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+pragma solidity ^0.8.20;
+
+contract Groth16Verifier {
```

Comment:
is this verifier contract with vkey on GitHub somewhere or did you generate it?

Reply:
I think I got it from [zupass-faucet](https://github.com/crisgarner/zupass-faucet/blob/main/packages/hardhat/contracts/Groth16Verifier.sol)

Reply:
This is forked from buidl guild groth16verifier which is using the production zkeys https://github.com/BuidlGuidl/zupass-scaffold-eth-2/blob/main/packages/hardhat/contracts/Groth16Verifier.sol